contracts/StrategyCurve3crv.sol

```diff
@@ -224,43 +234,16 @@ contract StrategyCurve3crv is StrategyCurveBase {
             uint256 _debtPayment
         )
     {
-        // harvest our rewards from the gauge
-        gauge.claim_rewards();
-        uint256 crvBalance = crv.balanceOf(address(this));
-        uint256 gnoBalance = gno.balanceOf(address(this));
-        // if we claimed any CRV, then sell it
-        if (crvBalance > 0) {
-            // keep some of our CRV to increase our boost
-            uint256 sendToVoter = crvBalance.mul(keepCRV).div(FEE_DENOMINATOR);
-            if (keepCRV > 0) {
-                crv.safeTransfer(voter, sendToVoter);
-            }
+        require(tradeFactory != address(0), "!yswaps");
 
-            // check our balance again after transferring some crv to our voter
-            crvBalance = crv.balanceOf(address(this));
+        // claim rewards from the gauge
+        // TODO: why continue claiming rewards within harvest if we no longer swap them here?
```

Comment:
If what we claim we cannot sell synchronously, why even bother claiming rewards here?

Reply:
interesting question. Do you believe keepers will call `claim_rewards()` separately?
I would leave the claim in harvest for legacy purposes.

Reply:
Yes, once ySwaps is incorporated as the standard I think it makes sense to come up with a new function in BaseStrategy that will let strategists take care of the claiming process or whatever process is applicable to the strategy. Then keepers can call this function. I understand why we might want to keep it this way for now. I'll note this in comments